ospfd/ospf_vty.c

```diff
@@ -2511,12 +2511,12 @@ DEFUN (no_ospf_refresh_timer,
 }
 
 
-DEFUN (ospf_auto_cost_reference_bandwidth,
-       ospf_auto_cost_reference_bandwidth_cmd,
-       "auto-cost reference-bandwidth (1-4294967)",
-       "Calculate OSPF interface cost according to bandwidth\n"
-       "Use reference bandwidth method to assign OSPF cost\n"
-       "The reference bandwidth in terms of Mbits per second\n")
+DEFUN(ospf_auto_cost_reference_bandwidth,
+      ospf_auto_cost_reference_bandwidth_cmd,
+      "auto-cost reference-bandwidth (1-4294967295)",
```

Comment:
I don't think this is necessary. The current range is in line with the range defined in the IETF OSPF YANG model:
```yang
      leaf reference-bandwidth {
        when "../enable = 'true'" {
          description "Only when auto cost is enabled";
        }
        type uint32 {
          range "1..4294967";
        }
        units Mbits;
        description
          "Configure reference bandwidth used to automatically
           determine interface cost (Mbits). The cost is the
           reference bandwidth divided by the interface speed
           with 1 being the minimum cost.";
      }
```